Mathlib/MeasureTheory/Measure/Typeclasses.lean

```diff
@@ -494,7 +494,39 @@ instance isFiniteMeasure_sFiniteSeq [h : SFinite μ] (n : ℕ) : IsFiniteMeasure
 lemma sum_sFiniteSeq (μ : Measure α) [h : SFinite μ] : sum (sFiniteSeq μ) = μ :=
```

Comment:
No, because most of the time it's used in the reverse direction: start from `μ`, break it into pieces, and argue separately on the pieces. You don't want `simp` to undo that.